app/controllers/product_controller.rb

```diff
@@ -3,4 +3,12 @@ class SlowFoodApp
   get '/' do
     erb :welcome
   end
+
+
+
+  get '/menu' do
+    @menu = Product.all
+    @menu_name = Menu.name
```

Comment:
I see no way this could work. What `Menu` are this referencing? 
Also, if you look at my comment above, and implement what I'm suggesting, you'll have access to the `name` attribute by calling `@menu.name`